sdk/monitor/monitor-opentelemetry-exporter/src/utils/metricUtils.ts

```diff
@@ -63,6 +74,23 @@ export function resourceMetricsToEnvelope(
           properties: {},
         };
         baseData.properties = createPropertiesFromMetricAttributes(dataPoint.attributes);
+
+        // If we're not exporting statsbeat, the metric is *not* a standard metric and the env var is set to false, we should not send the metric.
+        if (
+          shouldSendToOtlp() &&
+          isAksAttach() &&
+          !isStandardMetric(dataPoint) &&
+          process.env[ENV_APPLICATIONINSIGHTS_METRICS_TO_LOGANALYTICS_ENABLED] === "false" &&
```

Comment:
Do you want to support any other "case" `False` etc? or only lowercase?

Reply:
https://github.com/aep-health-and-standards/Telemetry-Collection-Spec/blob/main/ApplicationInsights/AKS_AutoAttach_Metrics.md#double-pumping-flag the spec here specifies "false" but I think at least allowing for case insensitivity is reasonable in this case.